networkx/generators/tests/test_inverse_line.py

```diff
@@ -0,0 +1,20 @@
+import networkx as nx
+from nose.tools import *
+
+from networkx.testing.utils import *
+# TODO
+# JC: What other tests would we like to see here?
```

Comment:
Other types of graphs. Empty graph. Inverse line graph of a graph that is not a line graph (to check an exception of the right type is raised)..